compose/compose.yml

```diff
@@ -1,7 +1,7 @@
 services:
   odyssey-timescale:
     container_name: odyssey-timescale
-    image: timescale/timescaledb:2.13.1-pg15
+    image: timescale/timescaledb-ha:pg15
```

Comment:
P sure I already merged in this change, will p go away after the merge conflicts are resolved)

Reply:
Alright.  Also we should try pg17 but perhaps seperate ticket.

Reply:
separate ticket sounds perfect